requirements.txt

```diff
@@ -1,5 +1,6 @@
 numpy
 nose
+pytest
```

Comment:
Pytest is in  travis/requirements.txt

Reply:
pycharm fault lawl il l'a ajouté automatiquement

Reply:
Mommy Pycharm knows best